test/functional/test_framework/key.py

```diff
@@ -1,226 +1,343 @@
-# Copyright (c) 2011 Sam Rushing
-"""ECC secp256k1 OpenSSL wrapper.
+# Copyright (c) 2019 Pieter Wuille
```

Comment:
Does the project have a policy on individual contributors' copyright notices in source files? Do you also want to include the MIT license boilerplate here too?

Reply:
There are a few files with individual author's names. As the "Copyright The Bitcoin Core contributors" is likely meaningless I think listing actual authors is preferable when it's a well-defined set of people, but I don't feel strongly.